.stylelintrc.json

```diff
@@ -1,19 +1,10 @@
 {
   "extends": ["stylelint-config-standard-scss"],
-  "plugins": [
-    "@ronilaukkarinen/stylelint-value-no-unknown-custom-properties",
```

Comment:
This could be removed from package.json, too.

Reply:
Noted.